### PR TITLE
Initial structure of the sponsor page

### PR DESCRIPTION
**Pull request by cmaureir**
2023-09-27-175956_1910x2150_scrot

### CODE REVIEW

File: generate_site/templates/sponsors-base.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+          mentorship, and aid to attend conferences and community events.
+        </p>
+      </div>
+    </div>
+    <div class="bg-pink mr-md-3 pt-3 px-3 pt-md-5 px-md-5 text-left overflow-hidden text-white">
+      <div class="my-3 p-3" id="the-organizers">
+        <h2 class="display-8">To learn more...</h2>
+        <p class="lead">
+          To learn more and to join us as a sponsor, please contact
+          sponsors@pyladies.com, and a member of our sponsorship team will
```
Comment (Mariatta) [inline generate_site/templates/sponsors-base.html:56]:
```suggestion
          [sponsors@pyladies.com](mailto:sponsors@pyladies.com), and a member of our sponsorship team will
```

File: generate_site/templates/sponsors-base.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          get back to you soon.
+        </p>
+        <p class="lead">
+          <a class="rarrow-link-white text-white" href="#">Get in touch</a>
```
Comment (Mariatta) [inline generate_site/templates/sponsors-base.html:60]: Should the href go somewhere? Or to the "sponsors@" email address?

**Review by Mariatta — CHANGES_REQUESTED** [review]
One of the URL is missing.

File: generate_site/templates/sponsors-base.html (hunk 2 above)
Comment (cmaureir) [reply generate_site/templates/sponsors-base.html:60]: yeah...in the meantime I guess the email will do.

Comment (cmaureir) [inline generate_site/templates/sponsors-base.html:60]:
```suggestion
          <a class="rarrow-link-white text-white" href="mailto:sponsors@pyladies.com">Get in touch</a>
```

**Comment by willingc** [comment]
@cmaureir It's so great to see this coming together. Thank you 💯

**Review by Mariatta — APPROVED** [review]
Thanks

**Pull request merged**